loadtest/bulkload.go

```diff
@@ -403,8 +403,11 @@ func GenerateBulkloadFile(config *LoadtestEnviromentConfig) GenerateBulkloadFile
 			for userInTeamNum := 0; userInTeamNum < numUsersToAddChannel; userInTeamNum++ {
 				userNum := usersInTeam[usersInTeamPermutation[userInTeamNum]]
 				userChannelImportData := &UserChannelImportData{
-					Name:  channel.Name,
-					Roles: "channel_user",
+					Name: channel.Name,
+				}
+				// Only assign roles if the channel isn't managed by a scheme
+				if channel.Scheme == "" {
+					userChannelImportData.Roles = "channel_user"
```

Comment:
I think this will result in API calls by that user to that channel not working, as they won't have any roles there. The fix that is needed is to add the `IsSchemeAdmin` and `IsSchemeUser` booleans to the userChannelImportData here (and for teams) both in the load tests and in the bulk importer.

I can make a PR to `mattermost-server` to fix this today.

Reply:
Yeah, I had noticed an increased "permission denied" rate that I was afraid was related to these changes. Thanks so much, @grundleborg!

Reply:
https://github.com/mattermost/mattermost-server/pull/9028 should fix this.

Reply:
Sweet -- I'll update this PR to leverage the correct fields.